src/cljs/nr/deckbuilder.cljs

```diff
@@ -850,6 +865,21 @@
    ;;    (tr [:deck-builder.create-game "Create Game"])])
    ])
 
+(defn view-toggles
+  [s deck]
+  [:div.decklist-view-options
+   [:h4 "View Options"]
+   (when (= (:side (:identity deck)) "Runner")
+     [:div
+      [:input {:type "checkbox" :checked (:show-mu-cost @s)
+               :on-change #(swap! s assoc :show-mu-cost (.. % -target -checked))}]
+      [:span "Show Memory Cost"]])
+   [:div
+    [:input {:type "checkbox" :checked (:show-credit-cost @s)
+             :on-change #(swap! s assoc :show-credit-cost (.. % -target -checked))}]
+    [:span "Show Credit Cost"]]
+   ])
```

Comment:
same

Reply:
i think this one is formatted proper, please advise further if not

Reply:
from:
```Clojure
  [:span ... ]]
])
```
to:
```Clojure
  [:span ... ]]])
```

Reply:
oh i got you thank you